lib/enum/nested.rb

```diff
@@ -18,15 +18,31 @@ def initialize(people_array)
   end
 
   def ages_sum
+    people.inject(0){|memo, num|
+    memo += num.age}
   end
 
   def average_age
+    people.inject(0){|memo, num|
+    memo += num.age} / 
+    people.map{|hash|
+      if hash.age
+        1
+      else
+        2
+      end}.count
```

Comment:
You may be interested in looking at the solutions file here to compare notes - https://github.com/ga-wdi-boston/wdi_10_ruby_enumerable/blob/solution/lib/enum/nested.rb